user/contactmethod/store.go

```diff
@@ -17,94 +19,13 @@ import (
 // Store implements the lookup and management of ContactMethods against a *sql.Store backend.
 type Store struct {
 	db *sql.DB
-
-	insert       *sql.Stmt
-	update       *sql.Stmt
-	delete       *sql.Stmt
-	findOne      *sql.Stmt
-	findOneUpd   *sql.Stmt
-	findMany     *sql.Stmt
-	findAll      *sql.Stmt
-	lookupUserID *sql.Stmt
-	enable       *sql.Stmt
-	disable      *sql.Stmt
-	metaTV       *sql.Stmt
-	setMetaTV    *sql.Stmt
-	now          *sql.Stmt
 }
 
 // NewStore will create a DB backend from a sql.DB. An error will be returned if statements fail to prepare.
 func NewStore(ctx context.Context, db *sql.DB) (*Store, error) {
 	p := &util.Prepare{DB: db, Ctx: ctx}
```

Comment:
```suggestion
func NewStore(ctx context.Context, db *sql.DB) *Store {
```
You can get rid of `p` and the error it might return